### PR TITLE
archive.md

### PR DESCRIPTION
**Pull request by piquark6046**
### To avoid invalid pull requests, please check and confirm following terms

- [X] This is not an ad/bug report;
- [X] My code follows the [guidelines](https://github.com/AdguardTeam/AdguardFilters/blob/master/CONTRIBUTING.md) and [syntax](https://kb.adguard.com/general/how-to-create-your-own-ad-filters) of this project;
- [X] I have performed a self-review of my own changes;
- [X] My changes do not break web sites, apps and files structure.

### If the problem does not fall under any category that is listed here, please write a comment below in corresponding section

- [ ] Missed ads or ad leftovers;
- [ ] Website or app doesn't work properly;
- [ ] AdGuard gets detected on a website;
- [X] Missed analytics or tracker;
- [ ] Social media buttons — share, like, tweet, etc;
- [ ] Annoyances — pop-ups, cookie warnings, etc;
- [ ] Filters maintenance.

### Enter the issue address

 - https://github.com/AdguardTeam/AdguardFilters/issues/137792

### Add your comment and screenshots

 - https://www.similarweb.com/website/archive.md/#traffic
 - https://www.similarweb.com/website/archive.is/#traffic
 - https://en.wikipedia.org/wiki/Archive.today

### Terms

- [X] By submitting this issue, I agree that pull request does not contain private info and all conditions are met

### CODE REVIEW

File: SpywareFilter/sections/specific.txt [hunk 1]
```diff
@@ -53,6 +53,7 @@
 /analytics.js$domain=freexcafe.com|rusvideos.tv|nakubani.ru|lun.com|songsara.net|newsmax.com|bolgehastanesi.com|ewtc.de|newsdig.tbs.co.jp|imore.com|city.kushiro.lg.jp|emeraldbridal.com.au|emarat-news.ae|imobie.jp
 ||assets.adobedtm.com^$domain=mcclatchydc.com|elnuevoherald.com|bellinghamherald.com|sacbee.com|charlotteobserver.com|newsobserver.com|miamiherald.com|ef.ru|algida.com.tr|bmw-motorrad.co.uk|cif.com.tr|clearhaircare.com|domestos.com|dove.com|elidor.com.tr|f1sport.autorevue.cz|geico.com|hellmanns.com.tr|herkessofraya.com|johnsonsbaby.com.tr|lipton.com|lnk.to|lux.com|magnumicecream.com|neutrogena.com.tr|omo.com|rexona.com|samsung.com|sap.com|schwarzkopf.com.tr|signalturkiye.com|msnbc.com|nbcnews.com|kbb.com|eurosport.ru|timgate.it|intermiles.com|oxfordmail.co.uk|spiegel.de|hp.com|deutschepost.de|thebay.com
 !
+||pixel.archive.*^$domain=~archive.org
```
Comment (Alex-302) [inline SpywareFilter/sections/specific.txt:56]: Why archive.org is excluded?
Do they have many mirrors?

Comment (piquark6046) [reply SpywareFilter/sections/specific.txt:56]: Please read the attached Wikipedia article.

Comment (Alex-302) [reply SpywareFilter/sections/specific.txt:56]: But your rule cannot break archive.org 
I don't understand.

Comment (piquark6046) [reply SpywareFilter/sections/specific.txt:56]: Because the owner is different, I excluded archive.org.
Do you want to include the website too?

Comment (Alex-302) [reply SpywareFilter/sections/specific.txt:56]: `$domain=~archive.org` just looks like a fix for some problem, but there is no problem) It's better to remove it so there's no confusion.

Comment (piquark6046) [reply SpywareFilter/sections/specific.txt]: Done.

**Review by Alex-302 — APPROVED** [review]

**Pull request merged**